packages/knip/src/plugins/jest/index.ts

```diff
@@ -16,7 +16,7 @@ const isEnabled: IsPluginEnabled = ({ dependencies, manifest }) =>
 
 const config = ['jest.config.{js,ts,mjs,cjs,json}', 'package.json'];
 
-const entry = ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'];
+const entry = ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)', '**/__mocks__/**'];
```

Comment:
Would be nice to have a file in that folder won't be reported as unused anymore

Reply:
I mean in one of the jest fixtures folders

Reply:
right, how can we achieve that?

what I did before was exclude them in `project`:

```
!/**/__mocks__/**
```

but I don’t think that’s what we’d generally want?

Reply:
We could an empty file such as `__mocks__/used.js` in of the fixtures folders. Before the change that was an unused file.

Btw, maybe also better to use something more specific like `**/__mocks__/**/*.{js,tx}`  - would that suffice?

Reply:
like this?

256aa4a9

Reply:
I see my own typo now 🤦‍♂️ but yeah that's fine (i wouldn't expect jsx to be necessary here)

Any chance you could still add that file?

Reply:
right, yeah, I think I got now what you mean: 615dd83a